### ISSUE TITLE
Use better semantic structure for the website

### ISSUE THREAD

**Issue by leouieda** [fatiando/website#34]
As @aguspesce rightly pointed out, the current website is mostly organised using `<div>` which is terrible for accessibility. Instead, we should be using `<section 
>` and `<article>` and be careful about placement of headings. See https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure

Any help with this would be more than welcome!
